packages/FormElement/src/components/Label/Label.js

```diff
@@ -4,56 +4,68 @@ import PropTypes from "prop-types";
 import useI18n from "@paprika/l10n/lib/useI18n";
 
 import labelStyles, { ruleStyles } from "./Label.styles";
+import Help from "../Help/Help";
```

Comment:
Still passing the help as a prop instead of just passing it as regular children to this component, purely because we have a specific order that we want the children to render in, ie the help should be to the right of the "optional" or "required" tag (which is rendered via prop control)